.pre-commit-config.yaml

```
@@ -58,7 +58,7 @@ repos:
   #     - id: check-manifest
 
   - repo: https://github.com/s-weigand/flake8-nb
-    rev: v0.3.1
+    rev: v0.4.0
     hooks:
       - id: flake8-nb
         additional_dependencies: ['pep8-naming']
```

```suggestion
        additional_dependencies: ['pep8-naming', 'packaging']
```

can we try this, as i tried on the container, it seems this error doesn't occur
however it occurring here, i m still investigating,anyhow this should work as well.